stl/inc/xutility

```diff
@@ -1025,7 +1025,11 @@ struct _Distance_unknown {
 };
 
 template <class _Diff>
+#ifdef __cpp_lib_concepts // TRANSITION, GH-395
+_INLINE_VAR constexpr _Diff _Max_possible_v{static_cast<_Make_unsigned_like_t<_Diff>>(-1) >> 1};
+#else // ^^^ handles integer-class types / integer types only vvv
 _INLINE_VAR constexpr _Diff _Max_possible_v{static_cast<make_unsigned_t<_Diff>>(-1) >> 1};
+#endif // ^^^ integer types only ^^^

```

Comment:
I suspect this won't be the last problem we have with `make_unsigned_t` vs. _`make-unsigned-like-t`_. Would it make more sense to push the problem down a layer by defining `_Make_unsigned_like_t` as an alias for `make_unsiged_t` in non-concepts mode instead of conditionally everything that we want to use `make_unsigned_t` / _`make-unsigned-like-t`_ in different language modes?

Reply:
Sure, I can validate and push changes.